### ISSUE TITLE
Creating an asset without checking-it to someone creates a server-error

### ISSUE THREAD

**Issue by Esya** [snipe/snipe-it#494]
To reproduce

* Create a new asset model
* Create a new asset on that model
* Don't checkout to anyone, set as ready to deploy
* Submit
* Try to go onto that asset's page

A line is added to `assets_logs` with `checkedout_to` equal to 0
```
mysql> SELECT * FROM asset_logs WHERE asset_id=14;
+----+---------+-------------+----------+---------------+-------------+---------------------+------------+------+----------+
| id | user_id | action_type | asset_id | checkedout_to | location_id | added_on            | asset_type | note | filename |
+----+---------+-------------+----------+---------------+-------------+---------------------+------------+------+----------+
| 17 |       1 | checkout    |       14 |             0 |        NULL | 2015-02-04 13:00:42 | hardware   |      | NULL     |
+----+---------+-------------+----------+---------------+-------------+---------------------+------------+------+----------+
1 row in set (0.00 sec)
```

Then when visiting that asset's page it tries to fetch that user (user 0) fullname which does not exist and makes it crash.

![img1](https://d1zjcuqflbd5k.cloudfront.net/files/acc_42229/1dzNx?response-content-disposition=inline;%20filename=Screenshot%20on%202015-02-04%20at%2017%3A34%3A29.png&Expires=1423051633&Signature=IPU8h5uZRPHNBbxJWjtHZGlznB5NgxovRLSi75RL4G~SGMrkUP9YZ9rk0j7yo7Ij-HjMbdW3j3OS42bYNpqCn7vyWEDX9rmhSfvXUGIFDepRUeQDkdWF6KfbM-N3S6EGvYQh3sVdEk7ip01KTVR-4rK4CPdhBpEBgcglvWL6lNo_&Key-Pair-Id=APKAJTEIOJM3LSMN33SA)

Am I doing something wrong?

**Comment by snipe** [comment]
Can you tell me what version or branch you're using?

**Comment by snipe** [comment]
(Also, if you have provide specific errors you're seeing, that would be helpful)

**Comment by cfinch** [comment]
I see this error with version 1.1. The asset is created properly, and you can edit it; it's just the view that's broken.

The error message is:

    Symfony \ Component \ Debug \ Exception \ FatalErrorException
    Call to a member function fullName() on null

open: /var/www/html/snipe/app/storage/views/79af58ac6b5f233e0eb53048e3c322da
```html
<?php if (isset($log->user_id)): ?> <?php echo e($log->adminlog->fullName()); ?>
<?php endif; ?>
</td>
<td><?php echo $log->action_type; ?></td>
<td>
<?php if (isset($log->checkedout_to)): ?>
<a href="<?php echo route('view/user', $log->checkedout_to); ?>">
<?php echo e($log->userlog->fullName()); ?>
</a>
<?php endif; ?>
```

**Comment by snipe** [comment]
This was fixed in 1.2.3 I believe
